images/docs-builder/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.redhat.io/ubi8
+MAINTAINER CodeReady Containers <devtools-cdk@redhat.com>
+
+RUN yum update -y
+RUN yum install -y ruby rubygems python38
```

Comment:
Better to do, it will only create single layer and also clean the yum cache so the end result of image should be lesser in size.

```
RUN yum update -y && \
    yum install --setopt=tsflags=nodocs -y \
    ruby rubygems python38 && \
    yum clean all && rm -rf /var/cache/yum/*
```

Reply:
Can we squash the layers somehow rather than having a hard to read line?

Reply:
Yeah, non trivial, have to look at how to squash the layers, but can be handled later imo, since this is an image we don't ship to the users, size shouldn't be that much of an issue.